wrap/.github/workflows/ci.yml

```diff
@@ -0,0 +1,52 @@
+name: Python CI
```

Comment:
Hmmm, did you merge in develop? Or is this a new file? I expected to see a diff

Reply:
Yes I merged develop, copied all relevant files and overwrite, tested and submitted this PR